source/tag-developers/set-tag.md

```diff
@@ -25,9 +25,10 @@ The scopes available are as follows:
 - `session` - the value will be set in session scope according to servlet spec. using the name as key
 - `request` - the value will be set in request scope according to servlet spec. using the name as key
 - `page` - the value will be set in page scope according to servlet spec. using the name as key
-- `action` - the value will be set in the request scope and Struts' action context using the name as key
+- `action` - the value will be set in the page scope and Struts' action context using the name as key
```

Comment:
I think action would be request (thread) scope theoretically?

Reply:
Not really, as you can chain actions so this will change which one is on the top of ValueStack.